### DISCUSSION TITLE
Choosing EDA analysis method & negative EDA

### DISCUSSION THREAD

**Discussion by ghost** [neuropsychology/NeuroKit#757] [Q&A] [answered]
Hello,

I'm using neurokit to extract a few measures of electrodermal activity for an experiment and I have a couple of questions.
1) There are different methods to a) clean the data, b) filter the signal into its tonic and phasic components and c) detect SCR peaks: what information do you typically use to drive the decision of which one to choose? 

2) It seems that the cleaned EDA signals (both using biopac and neurokit) contain negative values, which I don't think should be the case, same for the tonic signal (for both kim2004 highpass, median and neurokit highpass and median). I'm really not sure how to proceed from here, I assume it has to be with fine-tuning the filtering thresholds but I would really appreciate some guidance.

Thank you very much in advance for the help,
Assim

**Comment by DominiqueMakowski** [comment, marked as answer]
1) the pipeline should be informed based on your signal & your goals. For instance, one might want to use stronger filtering when expecting more noisy signals. Or, if the goal is to replicate the results from paper X, one might want to use the same pipeline.
2) That's usually not a problem: it usually indicates that the signal is somehow standardized (i.e., is not expressed in physical units but in relative units like for instance SD). That's usually not a problem (especially for comparing parts within the same signal), but again depends on what your goals are

**Reply by ghost** [reply]
Hi Dominique,

Thanks for the very quick reply. Both points make sense and are very helpful, thanks for this.
